python/ngen_cal/src/ngen/cal/ngen.py

```diff
@@ -408,8 +470,18 @@ def __init__(self, **kwargs):
             location = NWISLocation(nwis, nexus_data.name, nexus_data.geometry)
             nexus = Nexus(nexus_data.name, location, (), id)
             eval_nexus.append( nexus )
+        
+        if self.eval_feature:
+            for n in eval_nexus:
+                wb = self._flowpath_hydro_fabric[ self._flowpath_hydro_fabric['toid'] == n.id ]
+                print(wb)
```

Comment:
Seems like this was left in by mistake. Assume you were print debugging.

Reply:
Removed and rebased along with a branch update from gpkg-support